### PR TITLE
Project 1: Pull Request (FINAL)

### PR DESCRIPTION
**Pull request by Accelia**
Describe which aspect of this project you found most confusing or frustrating.
For me it was accepting that the project was not going to be "pretty" or look very good as it's just learning the basics of HTML and CSS. It's very hard for me to be happy w/ it only because it looks really basic but I know that's really what it has to be. 

Select which element of this project you are most proud of.
I'm the most proud of the bold text that emphasizes certain phrases I felt were really important, as well as the decorative bar under some of the text. I'm also pretty proud of using hsl colors.

Identify one thing you don’t currently know how to do in CSS that you’re interested to learn.
I want to know how people make really clean, graphic web pages honestly. I know we just started HTML + CSS but I want to learn how to do more complex things already because I'm pretty excited.

TO summarize the project requirements: you have to use at least 1 web font, have a background color, and a new text color. I know that I met these requirements and did more as well to practice HTML  + CSS

I think I mostly did the HTML and CSS standards of practice the only one I think I didn't really do was the alphabetical on CSS but I will apply It in the future.

### CODE REVIEW

**Comment by angeliquejw** [comment]
Link to Glitch project: https://glitch.com/~project-1-lnc

File: style.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+body {font-family: 'Caveat', cursive;
+  background-color: hsl(152, 56%, 65%);
+  color: hsl(152, 56%, 100%)
+}
+h1 {font-size: 40px;
+text-align: center;
+line-height: .5;
```
Comment (angeliquejw) [inline style.css:7]: Using line height values of less than 1 can cause issues down the road, so I want you to be aware of the implications -- this essentially sets the line height at 50% of the actual font size. I think you've done this to undo the default margins on your headings, but it's better to address the margins than to do this.

Here's a [short video](https://www.loom.com/share/83e302fa18ce4d2d877c053dbed1db77) that explains this more in depth, using your project and CSS. Let me know if you have further questions.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+color: hsl(315, 53%, 52%);}
+h2 {font-size: 27px;
+text-align: center;
+text-decoration: underline;
```
Comment (angeliquejw) [inline style.css:11]: 💡 On web pages, users generally associate underlined text with links -- so be thoughtful about how you use/apply underlines in your design!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+text-decoration-color: hsl(315, 79%, 67%);
```
Comment (angeliquejw) [inline style.css:12]: 👏🏻 Good job sussing out how to change the line color from matching the text color!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head> <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;700&display=swap" rel="stylesheet">
+    <meta charset="utf-8">
+    <title>My test page</title>
```
Comment (angeliquejw) [inline index.html:6]: Your `head` is missing a [proper `title` element](https://mica-web.github.io/learn/html/best-practices#include-a-proper-title-in-your-html-head). This is to add document information to browser windows and tabs and when a user bookmarks a site, among other things. It can include no other HTML or code, but it can include spaces, words or numbers.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head> <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;700&display=swap" rel="stylesheet">
```
Comment (angeliquejw) [inline index.html:4]: You call two font weights here (400, 700), but do not really use them in your CSS.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+line-height: .01;
+color: hsl(315, 53%, 52%);}
+p {font-size: 20px;
+line-height: 1.5;}
```
Comment (angeliquejw) [inline style.css:16]: Can you re-read and verify that you have:
1. the **Beautify** plugin enabled in VS Code?
1. followed the instructions for [configuring VS Code](https://mica-web.github.io/learn/tools/vs-code#configuring-your-text-editor)?

The nesting and lines inside your HTML and CSS are a bit wonkier than I expected, and both those things should clean things up as you work on files and save them.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+}
+h1 {font-size: 40px;
+text-align: center;
+line-height: .5;
+color: hsl(315, 53%, 52%);}
+h2 {font-size: 27px;
+text-align: center;
+text-decoration: underline;
+text-decoration-color: hsl(315, 79%, 67%);
+line-height: .01;
```
Comment (angeliquejw) [inline style.css:13]: See note about line height above.

**Review by angeliquejw — APPROVED** [review]
Great work here, Lauren -- See my note about `line-height` and a few other improvements, but I think you've got the basics down and are ready for Project 2!
